### DISCUSSION TITLE
GraphQL deprecation

### DISCUSSION THREAD

**Discussion by mrcheidel** [community/community#30749] [Other Feature Feedback, Questions, & Ideas]
Hi guys! 
What are the reasons to deprecate GraphQL? 

https://github.blog/changelog/2022-08-18-deprecation-notice-graphql-for-packages/

**Comment by desh-naik** [comment]
HI can someone from Github answer this question as i was about to use GraphQL API for my new project.  I would move to rest API if GraphQL will be deprecated permanently .. A quick response would be helpfull here
